config/jobs/kubernetes/sig-network/ci-e2e-gce-netd.yaml

```diff
@@ -547,10 +547,12 @@ periodics:
     containers:
     - args:
       - "--timeout=340"
-      - "--repo=k8s.io/kubernetes=release-1.11"
+      - "--repo=github.com/cadmuxe/kubernetes=release-1.11-calico-fix"
+      - --build=bazel
       - --scenario=kubernetes_e2e
       - --
-      - --extract=ci/latest
+      - --extract=local
+      - --stage=gs://kubernetes-release-pull/ci/ci-kubernetes-e2e-gce-netd-calico
```

Comment:
@krzyzacy I want to build and stage it to gs://kubernetes-release-pull/ci/ci-kubernetes-e2e-gce-netd-calico  and reuse it in the other testgrid. is it correct?

Reply:
just do `--extract=gs://kubernetes-release-pull/ci/ci-kubernetes-e2e-gce-netd-calico`, you are not staging anything here